framework/src/controller/validator/keywords/formatters.js

```diff
@@ -21,13 +21,20 @@ module.exports = {
 				const [ip, wsPort] = peer.split(':');
 				return {
 					ip,
-					wsPort: wsPort || 5000,
+					wsPort: parseInt(wsPort, 10) || 5000,
```

Comment:
Is this fixing another bug you found while fixing the whitelist issue?

Reply:
Kind of inconsistency not the issue. Somewhere sport was used as integers and while passed through command line it was considered as string.